bindata/network/ovn-kubernetes/microshift/master/daemonset.yaml

```diff
@@ -199,14 +199,15 @@ spec:
                 echo "$(date -Iseconds) - nbdb stopped"
                 rm -f /var/run/ovn/ovnnb_db.pid
         readinessProbe:
-          timeoutSeconds: 5
+          timeoutSeconds: 10
           exec:
             command:
             - /bin/bash
             - -c
             - |
               set -xeo pipefail
               /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/memory-trim-on-compaction on 2>/dev/null
+              /usr/bin/ovn-appctl -t /var/run/ovn/ovnnb_db.ctl --timeout=5 ovsdb-server/compact 2>/dev/null
```

Comment:
Do we want to also bump this timeout ?

Reply:
@maiqueb thanks for the review!
why this needs to be bumped? it may take more time when compacting database?

Reply:
I honestly do not know, I just saw you bumped the readiness probe timeout, and assumed we'd probably need to also extend this timeout. 

Disregard my comment if that is not the case.

Reply:
oh, I see your point. 
I bumped the overall timeout to 10s,  because by adding this commit it now has two cmds with each being 5s timeout.